tests/test_random_models.py

```diff
@@ -0,0 +1,56 @@
+#! /usr/bin/env python
+import os
+from copy import deepcopy
+
+# import matplotlib
+# matplotlib.use('TKAgg')
+import matplotlib.pyplot as plt
+import pytest
+import numpy as np
+import astropy.units as u
+
+import pint.models as tm
+from pint import fitter, toa
+from pinttestdata import datadir
+import pint.models.parameter as param
+from pint import ls
+from pint import utils
+
+
+def test_random_models():
+    # taken from test_fitter.py/test_fitter()
+    # Get model
+
+    m = tm.get_model(os.path.join(datadir, "NGC6440E.par"))
+
+    # Get TOAs
```

Comment:
I think the TOAs portion could be simplified considerably by using `get_TOAs()`, which does all of those later steps.  Or, alternatively, get both the model and the TOAs all at once (and in one line -- which is now how we should, in general, be recommending that people load things) using `pm.get_model_and_toas()`.

Reply:
I think I copied this from another test routine, but now I've updated it.  will push it up soon.